build.gradle

```diff
@@ -1,39 +1,18 @@
 plugins {
-  id 'com.gradleup.shadow' version '8.3.5'
-  id 'application'
-  id 'antlr'
+  id 'java'
 }
 
-java {
-  sourceCompatibility = JavaVersion.VERSION_17
-  targetCompatibility = JavaVersion.VERSION_17
-}
-
-repositories {
-  mavenCentral()
-}
+allprojects {
+  java {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
+  }
```

Comment:
This is a bit old style, now gradle allows sharing build logic using "convention" plugins. See here https://docs.gradle.org/current/samples/sample_convention_plugins.html#organizing_build_logic

see an example [here](https://github.com/seqeralabs/libseqera/blob/403ab8ac8813e79b6815df3b032feed0928a24fa/buildSrc/src/main/groovy/io.seqera.groovy-application-conventions.gradle#L14-L14) and usage [here](https://github.com/seqeralabs/libseqera/blob/403ab8ac8813e79b6815df3b032feed0928a24fa/lib-mail/build.gradle#L2-L3)

Reply:
done